examples/fpga/artya7-100/README.md

```diff
@@ -0,0 +1,46 @@
+# Ibex RISC-V Core SoC Example
+
+This is a simple example of how Ibex can be utilized.
+The core is not connected to a bus, but to the SRAM directly, which is initialized
+by the `led.vmem` file from the software section.
+
+## Requirements
+
+### Tools
+
+  - RISC-V compiler
+  - srecord
+  - `fusesoc` and its dependencies
+  - Xilinx Vivado
+
+## Build
+
+### Software
+
+First the software must be built. Go into `sw/led` and call:
+
+```
+make CC=/path/to/RISC-V-compiler
+```
+This should produce a `led.vmem` file which is used in the synthesises to update
+the SRAM storage.
+
+### Hardware
+
+Now run the command in the `example` folder:
+
+```
+fusesoc --cores-root=../ build lowrisc:ibex:top_artya7_100
+```
+
+This will create a directory `build` which contains the output files, including
+the bitstream.
+
+## Program
+
+Open the Vivado project:
+`build/lowrisc_ibex_top_artya7_100_0.1/synth-vivado/lowrisc_ibex_top_artya7_100_0.1.xpr`
+
+And use the Hardware Manager to program the Arty board:
+`build/lowrisc_ibex_top_artya7_100_0.1/synth-vivado/lowrisc_ibex_top_artya7_100_0.1.bit`
```

Comment:
`fusesoc pgm` should do the trick as well

Reply:
Thanks for that, much better to program this way.